sdk/servicebus/Azure.Messaging.ServiceBus/CHANGELOG.md

```diff
@@ -1,7 +1,18 @@
 # Release History
 
-## 7.0.0-preview.8 (Unreleased)
+## 7.0.0-preview.8 (2020-10-06)
 
+### Added
+- Added `AcceptSessionAsync` that accepts a specific session based on session ID.
+
+### Breaking Changes
+- Renamed `ViaQueueOrTopicName` to `TransactionQueueOrTopicName`.
+- Renamed `ViaPartitionKey` to `TransactionPartitionKey`.
+- Renamed `ViaEntityPath` to `TransactionEntityPath`.
+- Renamed `Proxy` to `WebProxy`.
+- Made `MaxReceiveWaitTime` in `ServiceBusProcessorOptions` and `ServiceBusSenderOptions` internal.
```

Comment:
ServiceBusSenderOptions? I don't think this was ever there.

Reply:
Fixed.